src/importexport/musicxml/tests/tst_mxml_io.cpp

```diff
@@ -262,6 +264,17 @@ static void fixupScore(Score* score)
     score->setSaved(false);
 }
 
+static void checkScore(Score* score)
+{
+    if (score->parts().first()->nstaves() > 1) {
+        Measure* measure = score->lastMeasure();
+        Segment* segment = measure->findSegment(SegmentType::EndBarLine, measure->endTick());
+        BarLine* barline = (BarLine*)(segment->element(0));
+        bool span = barline->spanStaff();
+        QVERIFY(barline->spanStaff());
+    }
+}
+
 void TestMxmlIO::setValue(const std::string& key, const Val& value)
```

Comment:
I wonder where the comment went that the 3.x branch has here?

    //---------------------------------------------------------
    //   mxmlIoTest
    //   read a MusicXML file, write to a new file and verify both files are identical
    //---------------------------------------------------------

Reply:
It's sill there?